### PR TITLE
NO-ISSUE: fix lvm compact resources

### PR DESCRIPTION
**Pull request by eifrach**

### CODE REVIEW

**Comment by eifrach** [comment]
/pj-rehearse pull-ci-openshift-assisted-service-master-edge-e2e-metal-assisted-lvm
/pj-rehearse auto-ack

**Comment by eifrach** [comment]
/pj-rehearse pull-ci-openshift-assisted-service-master-edge-e2e-metal-assisted-lvm

File: ci-operator/config/openshift/assisted-installer/openshift-assisted-installer-master__edge.yaml [hunk 1]
```diff
@@ -224,7 +224,7 @@ tests:
         OLM_OPERATORS=lvm
         NUM_MASTERS=3
         NUM_WORKERS=0
-        MASTER_MEMORY=18000
+        MASTER_MEMORY=25600
```
Comment (rccrdpccl) [inline ci-operator/config/openshift/assisted-installer/openshift-assisted-installer-master__edge.yaml:227]: would this fit without changing machine type?

Comment (rccrdpccl) [reply ci-operator/config/openshift/assisted-installer/openshift-assisted-installer-master__edge.yaml:227]: self-answer: if it wouldn't, the test wouldn't have passed :sweat_smile:

**Pull request merged**